bootstrap/bootstrapper.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: kubeflow-cluster-admin
+subjects:
+  - kind: ServiceAccount
+    # Reference to upper's `metadata.name`
+    name: default
+    # Reference to upper's `metadata.namespace`
+    namespace: default
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: rbac.authorization.k8s.io
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: kubeflow-bootstrapper
+spec:
+  clusterIP: "None"
+---
+apiVersion: v1
```

Comment:
Unless you're creating a PersistentVolume ahead of time I had to remove the PersistentVolumeClaim and add a volumeClaimTemplates section to StatefulSet - see below